main/main.go

```diff
@@ -162,11 +164,23 @@ func reportExecutionStatus() {
 
 func install() {
 	operation := "install"
+	path, _ := os.Getwd()
+	_, err := exec.Command("/bin/sh", filepath.Join(path, "scripts/service_install.sh")).Output()
+	if err != nil {
+		errorMessage := fmt.Sprintf("Error installing service: %+v", err)
+		errorLogger.Println(errorMessage)
```

Comment:
This will just print the error, it wont fail the operation. Is that the expected behavior? To fail the operation you should add this line here - 

`executionErrors = append(executionErrors, errorMessage)`